lib/src/model/unit.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:meta/meta.dart';
+
+@immutable
+class Unit {
```

Comment:
It's a pity every library needs its own `Unit` (like in [sum_types](https://github.com/werediver/sum_types.dart/blob/19f96753f76c6e9e9f93e8d765d900dded9e6a5e/sum_types/lib/src/support.dart#L4) and now here). But we can rightfully blame the poorly designed Dart SDK 🎉